src/components/Resource.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { TResource } from '../types';
+interface Props {
+  id: string;
+  resource: TResource;
+}
+
+const Resource = (props: Props) => {
+  const {
+    resource: { charityname }
```

Comment:
https://media.giphy.com/media/cLtvKDcQYkSRO/giphy.gif

Reply:
not a fan of that deep of destructuring i take it? 😛